src/nifc/typenav.nim

```diff
@@ -41,7 +41,7 @@ proc getTypeImpl(m: var Module; n: Cursor): Cursor =
       result = getTypeImpl(m, m.src.cursorAt(d.pos))
     else:
       # importC types are not defined
-      result = createIntegralType(m, "(err)")
+      result = n
```

Comment:
This whole thing computes the **Type** of an **expression**. For example for `x` when `x` is `var x: int` it returns `int`. The expression is never the type itself, ergo `result = n` is simply wrong and `createIntegralType(m, "(err)")` was correct, indicating we could not compute the type.

Reply:
Well that's an existing bug then, but don't make it worse. ;-)